### DISCUSSION TITLE
Pulling image out to use as standalone installer

### DISCUSSION THREAD

**Discussion by inor0276** [sickcodes/Docker-OSX#237] [Q&A]
Is it possible to pull this docker image out & create a standalone installer for the same PC?
--Please be kind, I'm a noob.

**Comment by sickcodes** [comment]
Yeah it sure is,

Copy image out using:
https://github.com/sickcodes/Docker-OSX#quick-start-your-own-image-naked-container-image



Pull images out using `sudo find /var/lib/docker -size +10G | grep mac_hdd_ng.img` or  `docker cp`


And then follow https://sick.codes/how-to-install-macos-virtual-machine-on-linux-arch-manjaro-catalina-mojave-or-high-sierra-xcode-working/

Or follow the guides on https://github.com/kholia/osx-kvm

**Reply by inor0276** [reply]
Thanks a lot. I will try it out.
Your Great work is appreciated.
